static/app/components/orgRole.tsx

```diff
@@ -0,0 +1,123 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ExternalLink from 'sentry/components/links/externalLink';
+import Link from 'sentry/components/links/link';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconInfo} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Member, Organization} from 'sentry/types';
+import {sortOrgRoles} from 'sentry/utils/orgRole';
+
+export const OrgRoleInfo = ({
+  organization,
+  member,
+}: {
+  member: Member;
+  organization: Organization;
+}) => {
+  const {orgRoleList} = organization;
+  const {orgRole, orgRolesFromTeams} = member;
+
+  const orgRoleFromMember = orgRoleList.find(r => r.id === orgRole);
+  if (!orgRoleFromMember) {
+    throw new Error();
+  }
+
+  if (!orgRolesFromTeams || orgRolesFromTeams.length === 0) {
+    return <Fragment>{orgRoleFromMember.name}</Fragment>;
+  }
+
+  const topOrgRoleId = sortOrgRoles(
+    orgRolesFromTeams.map(r => r.role.id).concat([orgRoleFromMember.id]),
+    orgRoleList
+  )[0];
+  const topOrgRole = orgRoleList.find(r => r.id === topOrgRoleId);
+  if (!topOrgRole) {
+    throw new Error();
+  }
+
+  const urlPrefix = `/settings/${organization.slug}/`;
+
+  const node = (
+    <RoleTooltip>
+      <div>{t('This user recieved org-level roles from several sources.')}</div>
+
+      <ListWrapper>
+        <TeamRow>
+          <TeamLink to={`${urlPrefix}member/${member.id}/`}>
+            {t('User-specific')}
+          </TeamLink>
+          <div>: {orgRoleFromMember.name}</div>
+        </TeamRow>
+
+        <br />
+
+        <div>{t('Teams:')}</div>
+        {orgRolesFromTeams
+          .sort((a, b) => a.teamSlug.localeCompare(b.teamSlug))
+          .map(r => (
+            <TeamRow key={r.teamSlug}>
+              <TeamLink to={`${urlPrefix}teams/${r.teamSlug}/`}>#{r.teamSlug}</TeamLink>
+              <div>: {r.role.name}</div>
+            </TeamRow>
+          ))}
+      </ListWrapper>
+
+      <div>
+        {t(
+          'Sentry will grant them permissions equivalent to their highest org-level role.'
```

Comment:
```suggestion
          'Sentry will grant them permissions equivalent to their highest priority org-level role.'
```

perhaps we should explicitly mention this priority in the docs since the role are only _listed_ in priority order there

Reply:
I'll change this to "union set of permissions of all their org-level roles"